docker-compose.yml

```diff
@@ -14,6 +14,13 @@ services:
       - "redis"
       - "zookeeper"
       - "redis-cluster-node"
+  test-baseplate:
+    build:
+      context: "."
+      dockerfile: "Dockerfile"
+    command: "pytest -v tests/"
+    volumes:
+      - ".:/src"
```

Comment:
happy to remove this if we dont want it, but I found it helpful for testing locally (I'm not sure how most people run these tests?)